src/token.ts

```diff
@@ -81,19 +82,35 @@ export class WriteableToken extends ReadOnlyToken {
   }
 
   async transferToNewEscrow(beneficiary: EthereumAddress, holder: EthereumAddress) {
-    trace(`Deploying escrow contract before transferring`);
+    const tokenOwner = await this.getOwner();
+    if (tokenOwner instanceof WriteableTitleEscrowOwner) {
+      trace(`Current Owner ${tokenOwner.address} is a Title Escrow, calling deployAndTransfer`);
+      return tokenOwner.deployAndTransfer(beneficiary, holder);
+    }
+    trace(`Current Owner ${tokenOwner.address} is a Ethereum Account, deploying escrow contract before transferring`);
     const escrowInstance = await WriteableTitleEscrowOwner.deployEscrowContract({
```

Comment:
Hmm still two transactions here. Wondering if the deployment should be in the ERC721 so that it transfer and deploy, but i don think we should mess around more with that contract. 

But if we do... then the ERC721 can implement `deployAndTransferTitleEscrow` which does what it says only the owner can execute, and from the escrow POV, it checks for authorization and then call the registry's deploy and transfer instead of implementing it... another way to look at it

Reply:
i thought about that but that's a lot of logic in the ERC721..... and i got lazy so i just left this 2-step here because it doesn't show up in the metamask workflow anyway, this part is done under config-dapp which doesn't prompt for confirms

Reply:
hmm okay, would you rather take out this section and mark it as unsupported? Just so that we dont leave this for someone else or ourselves to discover now that we are told that it's always a contract owning it and not an EOA?